easybuild/easyconfigs/s/STAMP/STAMP-1.3-intel-2016a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'CmdCp'
+
+name = 'STAMP'
+version = '1.3'
+
+homepage = 'http://www.benoslab.pitt.edu/stamp/'
+description = """STAMP is a tool for characterizing similarities between transcription factor binding motifs"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['https://github.com/seqcode/stamp/archive/']
+sources = ['v%(version)s.tar.gz']
+
+dependencies = [('GSL', '2.1')]
+
+start_dir = 'src'
+
+cmds_map = [('.*', "$CXX $CXXFLAGS -o stamp *.cpp -lm -lgsl -lgslcblas")]
```

Comment:
@wpoely86 Why? To ensure GSL is picked up from the right location? That's taken care of via `$LIBRARY_PATH`, no?